_layouts/default.html

```diff
@@ -18,6 +18,8 @@
 <div class="container">
 
     <div id="navigation">
+        <!-- Integrated Searh Box from docdoc Go -->
+<iframe src="https://duckduckgo.com/search.html?width=300&site=https://www.opus-repository.org/&prefill=Search &bgcolor=00000" style="overflow:hidden;margin-right:0;padding:0;width:258px;height:40px;" frameborder="0"> </iframe> 
```

Comment:
Lager das Styling mal bitte in die Datei `css/style.css` aus. Das Style-Attribut sollte nur sehr selten verwendet werden.

Reply:
Hab ich gemacht. Hat etwas gedauert bis begriff, wie es geht, aber es hat geklappt.